fearless/Common/DataProvider/Subscription/WalletLocalStorageSubscriber.swift

```diff
@@ -57,6 +63,64 @@ extension WalletLocalStorageSubscriber {
 
         return accountInfoProvider
     }
+
+    func subscribeToOrmlAccountInfoProvider(
+        for accountId: AccountId,
+        chain: ChainModel
+    ) -> AnyDataProvider<DecodedOrmlAccountInfo>? {
+        guard let accountInfoProvider = try? walletLocalSubscriptionFactory.getOrmlAccountProvider(
+            for: accountId,
+            chain: chain
+        ) else {
+            return nil
+        }
+
+        let updateClosure = { [weak self] (changes: [DataProviderChange<DecodedOrmlAccountInfo>]) in
+            guard let ormlAccountInfo = changes.reduceToLastChange()?.item else {
+                return
+            }
+            let accountInfo = AccountInfo(
+                nonce: 0,
+                consumers: 0,
+                providers: 0,
+                data: AccountData(
```

Comment:
Maybe add additional constructor for `AccountData` which accepts `OrmlAccountInfo`?